hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/table/ITTestHoodieDataSource.java

```diff
@@ -1303,8 +1303,8 @@ void testParquetComplexTypes(String operation) {
         () -> tableEnv.sqlQuery("select * from t1").execute().collect());
     final String expected = "["
         + "+I[1, [abc1, def1], {abc1=1, def1=3}, +I[1, abc1]], "
-        + "+I[2, [abc2, def2], {def2=3, abc2=1}, +I[2, abc2]], "
-        + "+I[3, [abc3, def3], {def3=3, abc3=1}, +I[3, abc3]]]";
+        + "+I[2, [abc2, def2], {def2=3, abc2=1}, +I[2, null]], "
+        + "+I[3, [abc3, def3], {def3=3, abc3=1}, null]]";
```

Comment:
Why outputs nulls here ?

Reply:
@danny0405, the output `+I[2, null]` is used to verify whether the return value is non-null value for part of child columns are null, meanwhile the output `null` is used to verify whether the return value is null value of all child columns are null.

Reply:
@danny0405, this change is aimed to verify whether the value of the row type column isn't null value when the value certain child columns is null.